src/bin/swcci.rs

```diff
@@ -0,0 +1,12 @@
+mod repl;
+
+fn main() {
+    let mut repl = repl::Repl::new();
+    match repl.run() {
+        Ok(_) => {}
+        Err(err) => {
+            println!("Unknown error in the REPL occurred: {}", err);
```

Comment:
This seems a little verbose:
```suggestion
            println!("error: {}", err);
```